frontend/src/components/Home.vue

```diff
@@ -166,10 +260,14 @@ export default {
         this.portfolioTotal = msg.portfolioTotal
         this.referralTotal = msg.referralTotal
         this.assets = msg.assets
-
         this.minChangePercent = 0
         this.maxChangePercent = 0
+        this.totalProfit = 0
         msg.assets.forEach((asset) => {
+          this.totalProfit += 
+          this.isrothree = asset.shares < 3 && !this.isRainbow(asset)
```

Comment:
`isrothree` indicates whether or not a stock ticker should be highlighted with indigo to show that it is a candidate for selling/buying to the 3 shares or 0 shares. Not the best variable name...